packages/babel-preset-mc-app/index.js

```diff
@@ -60,18 +60,14 @@ module.exports = function getBabePresetConfigForMcApp(api, opts = {}) {
           // Will use the native built-in instead of trying to polyfill
           // behavior for any plugins that require one.
           ...(opts.runtime !== 'automatic'
-            ? { useBuiltIns: true, pragmaFrag: 'Fragment' }
+            ? {
+                useBuiltIns: true,
+                importSource: '@emotion/core',
```

Comment:
![image](https://user-images.githubusercontent.com/1110551/98484682-8f5da700-2211-11eb-8c94-544b78e5a31e.jpeg)

If I understand this correctly, we should always use this option, right? 🤔

Reply:
I think so. I read the changelogs of most emotion packages and find things quite interestingly confusing. I think we can always use this yes. 

Then however I am not sure why the build fails with the usual 'Can not find React'. Something doesn't kick in. Maybe an outdated version though.